cmd/gaia/init/testnet.go

```diff
@@ -182,67 +200,115 @@ func testnetWithConfig(config *cfg.Config, cdc *codec.Codec) error {
 			return err
 		}
 
-		// Gather gentxs folder
+		// gather gentxs folder
 		err = writeFile(fmt.Sprintf("%v.json", nodeDirName), gentxsDir, txBytes)
 		if err != nil {
 			_ = os.RemoveAll(outDir)
 			return err
 		}
 	}
 
-	// Generate empty genesis.json
+	if err := initGenFiles(cdc, chainID, accs, genFiles); err != nil {
+		return err
+	}
+
+	if err := collectGenFiles(cdc, config, chainID, monikers, nodeIDs, valPubKeys); err != nil {
+		return err
+	}
+
+	fmt.Printf("Successfully initialized %d node directories\n", numValidators)
+	return nil
+}
+
+func initGenFiles(
+	cdc *codec.Codec, chainID string, accs []app.GenesisAccount, genFiles []string,
+) error {
```

Comment:
this is a weird way to split up the `func` header, I'd suggest something along the lines of: 
```
func initGenFiles(cdc *codec.Codec, chainID string,
     accs []app.GenesisAccount, genFiles []string) error {
```

Reply:
Ughhh I find that so much less elegant and harder to read imho. I'll be happy to change it if it's our standard though.

Reply:
What's prettier:

a.
```golang
foo := []string{"foo",
	"bar", "cat"}
```

b.
```golang
foo := []string{
	"foo", "bar", "cat",
}
```

Reply:
k - yeah so-far the style I've suggested is used everywhere, so I think you should change it - we could open up a stylistic issue to discuss further (but post-launch change obv.).
IMO the suggested change is more elegant! haha it's stylistic 4sure though 

I think functions are unique from struct definition, I don't find it adequate to draw the comparison (but ftr obv. `b.`)

Reply:
Ok, I'm vetoed here haha. I'll change it. Will open up an issue later to discuss this more openly.